### PR TITLE
Zuora AQuA User Guide

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

**Review by sorinmare — APPROVED** [review]
Zuora AQuA Conector is a smart connector generated using Rest Connect 2.0
Maybe we should mention that in zuora-aqua-connector.adoc (or some other place that seems suitable like release notes)

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -358,6 +358,12 @@
  ** xref:xml/xml-xslt.adoc[XSLT Transformations with the XML Module]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc:358]: INSTRUCTIONS FOR EDIT:

Can you please edit this PR by EOD Monday? Please do NOT edit the zuora-aqua-connector-reference file (generated reference). The rest of the files are generated from the connector user guide template, and you have edited that content before. Very little of it is specific to this connector. The two files to concentrate on would be: zuora-aqua-connector-examples and zuora-aqua-connector.

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+== Example: Retrieve Details for an Account
+
+This example retrieves the account details for the specified account ID and returns them in JSON format.
+
+== Studio Flows
+
+* Listener (HTTP) +
+Accepts data from HTTP requests
+* Zuora API Connector +
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-examples.adoc]: Consider using full official connector name at first use per .adoc.

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+* From the Exchange button in the Studio taskbar
+* From the Mule Palette view
+
+=== Add the Connector Using Exchange
+
+. In Studio, create a Mule Project.
+. Click the Exchange *(X)* icon in the upper-left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for *Zuora AQuA*.
+. Select the connector, and click *Add to project*.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc]:
```suggestion
. Select the connector and click *Add to project*.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X)* Search in Exchange*.
+. In *Add Modules to Project*, type "Zuora AQuA" in the search field.
+. Click this connector's name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source
+
+Configure an input source for the connector such as an operation for this connector, *HTTP Listener*, or *Scheduler*.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc]:
```suggestion
Configure an input source for the connector, such as an operation for this connector, *HTTP Listener*, or *Scheduler*.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+== Configure the Connector
+
+To configure this connector:
+
+. Drag and drop a connector operation into the Studio canvas.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc]:
```suggestion
. Drag a connector operation into the Studio canvas.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+. To create a global element for the connector, set these fields:
++
+** *Username*: Zuora username.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc]:
```suggestion
** *Username*: Zuora username
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+** *Password*: Zuora password.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-studio.adoc]:
```suggestion
** *Password*: Zuora password
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-xml-maven.adoc [hunk 8]
```diff
@@ -0,0 +1,68 @@
+= Zuora AQuA XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to your XML Mule flow and in the Apache Maven POM file.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-xml-maven.adoc]:
```suggestion
If you manually code a Mule app in XML, either from the Anypoint Studio XML editor or from a text editor, you can access the connector both in your app, by adding reference statements to your XML Mule flow, and in the Apache Maven POM file.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-xml-maven.adoc [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+[NOTE]
+====
+You can avoid the manual process described in this topic by using Studio. When you install the connector in Studio, it automatically populates the XML code with the connector's namespace and schema location, and adds a dependency to the POM file.
+====
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-xml-maven.adoc]:
```suggestion
Paste the following code inside the `<mule>` tag of the header of your configuration XML:
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector.adoc [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+|Anypoint Studio |7.1.2 and later
+|Zuora |R249
+|===
+
+== Audience
+
+* Starting user
++
+To create your Mule app, see xref:zuora-aqua/zuora-aqua-connector-design-center.adoc[Design Center] or
+xref:zuora-aqua/zuora-aqua-connector-studio.adoc[Anypoint Studio]. The [Additional Configuration Information] topic helps extend your understanding of the Zuora AQuA connector. The [Examples] topic provides one or more use cases for using this connector.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector.adoc]:
```suggestion
xref:zuora-aqua/zuora-aqua-connector-studio.adoc[Anypoint Studio]. The [Additional Configuration Information] topic helps extend your understanding of Zuora AQuA Connector. The [Examples] topic provides one or more use cases for using this connector.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector.adoc [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
++
+* Power user
++
+Read the [xref:zuora-aqua/zuora-aqua-connector-xml-maven.adoc[XML and Maven Support] and xref:zuora-aqua/zuora-aqua-connector-examples.adoc[Examples] topics.
+
+== Common Use Cases for this Connector
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector.adoc]:
```suggestion
== Common Use Cases for This Connector
```

Comment (kbostrom) [reply modules/ROOT/pages/zuora-aqua/zuora-aqua-connector.adoc]: No common uses?

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc]:
```suggestion
. Specify a value for *Project name* and click *Create*.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the trigger card.
+. Search for and select *HTTP Listener* or *Scheduler*.
+. Complete the fields with the values you want to configure for your trigger. For more information on field values, see the xref:zuora-aqua/zuora-aqua-connector-reference.adoc[Zuora AQuA Connector Reference - Mule 4].
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc]:
```suggestion
. Complete the fields with the values you want to configure for your trigger. 
For more information about field values, see the xref:zuora-aqua/zuora-aqua-connector-reference.adoc[Zuora AQuA Connector Reference - Mule 4].
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+== Configure the Zuora AQuA Target Component
+
+To add a Zuora AQuA connector as a component:
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc:27]:
```suggestion
To add Zuora AQuA Connector as a component:
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the connector name.
+. Select the connector.
+. Select an operation for the connector.
+. If needed, specify values that are required in the *General* tab.
++
+For example, if you choose *Add Attachments* as the operation, you must enter a value for the *Associated Object Type* and *Associated Object Key fields*.
++
+. Click *Click here to set it up* to set the required fields.
+. Specify access information to the connector resource. For this connector, you can use only basic authentication, as described below.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc]:
```suggestion
. Specify access information to the connector resource. 
For this connector, you can use only basic authentication, as described in Parameters for the Basic Auth Configuration.
```

File: modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+|*Password*| Zuora password
+|*Base URI* a| Base path of the AQuA API for your instance. For example: + 
+`https://apisandbox-api.zuora.com:443/`
+|===
+
+image::zuora-aqua/aqua-config.png[Zuora API Configuration]
+
+== Next Step
+
+After creating an app in Flow Designer, see the xref:zuora-aqua/zuora-aqua-connector-examples.adoc[Zuora AQuA Examples] section to experiment with the Zuora AQuA connector.
```
Comment (kbostrom) [inline modules/ROOT/pages/zuora-aqua/zuora-aqua-connector-design-center.adoc]:
```suggestion
After creating an app in Flow Designer, see the xref:zuora-aqua/zuora-aqua-connector-examples.adoc[Zuora Examples] section to experiment with Zuora AQuA Connector.
```

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**